README.md

```diff
@@ -337,9 +337,9 @@ A small `config.js` file at the root of the project provides basic path settings
 
 [Webpack](https://webpack.js.org/) does the heavy lifting of assets compilation and transformation. Webpack allows importing of Sass, images, fonts, and other assets into javascript files, bundling output files like CSS and javascript. There are three webpack config files:
 
-- `webpack.shared.config.js`: The majority of loader and plugin configuration related to `source/`
-- `webpack.drupal.config.js`: Configuration specific to `app/drupal`
-- `webpack.pl.config.js`: Configuration specific to `app/pl`
+- `webpack.particle.dev.js`: The majority of loader and plugin configuration related to `source/`
+- `webpack.drupal.prod.js`: Configuration specific to `app/drupal`
+- `webpack.pl.dev.js`: Configuration specific to `app/pl`
```

Comment:
Blarg, gotta update docs with our new structure. I can hit this today.

Reply:
Yeah, these were just auto-replaced from PHPstorm. but.... this entire readme [minus these name updates] has already been copied to the front end docs repo *cough*